lib/workers/package-file/package.js

```diff
@@ -16,6 +16,10 @@ async function renovatePackage(config) {
     logger.debug('package is disabled');
     return [];
   }
+  if (config.packageVersionMatches === false) {
```

Comment:
What is the purpose of this particular option? It seems to behave the exact same way as `config.enabled` on line 15 above.

Reply:
All this should be removed actually - the logic for this should be contained within the packageRules evaluation